integration/src/index.tsx

```diff
@@ -10,7 +10,7 @@ import {
 
 ReactDOM.render(
     <div id="classic-accordion">
-        <Accordion>
+        <Accordion allowMultipleExpanded={false} allowZeroExpanded={false}>
```

Comment:
It was either update the tests or update the accordion, and I chose the lazy option 😬 

Fair call, I will take a look.

Reply:
Hmm, if I change it to run against the default settings, the integration test fails (as I expected). 

However, the failing integration test in question is written assuming the old default behaviour. Whereas the other tests don't – they just test based on the DOM they see (e.g. if an accordion is `aria-expanded`, the panel should be visible). The integration tests run in a browser (with puppeteer?).

The failing test opens the first panel, then expects that it can't be collapsed (because it's the only one open, and `allowZeroExpanded:false` _was_ the default behaviour).

RAA doesn't add any DOM cues regarding `allowMultipleExpanded` or `allowMultipleExpanded`, so we can't really test against them.

I think the best course of action here is to remove that particular test. We already have an integration test that opening a panel works. 

The alternative is adding more DOM guff like `data-disallow-multiple-expanded`/`data-disallow-zero-expanded`, but that feels like a too-big change to me.

Reply:
(done)